app/models/ontology/sentences.rb

```diff
@@ -1,135 +1,151 @@
+            mapping.symbol_mappings)
+        else
+          default_translated_sentences
+        end
       end
-    end
 
-    protected
-    def translate_sentences_for(audience_ontology)
-      link = Link.where(source_id: audience_ontology, target_id: self, kind: 'import').first
-      if link && link.entity_mappings.any?
-        create_or_fetch_translations(audience_ontology, link, link.entity_mappings)
-      else
-        default_translated_sentences
+      # If there are no translations applicable, what do you do?
+      def default_translated_sentences
+        self.sentences
       end
-    end
 
-    # If there are no translations applicable, what do you do?
-    def default_translated_sentences
-      self.sentences
-    end
+      def create_or_fetch_translations(audience_ontology, mapping, mappings,
+        overwrite: false)
+        translations =
+          TranslatedSentence.for(audience_ontology, sentences_from: self)
+        if translations.any? && !overwrite
+          translations
+        else
+          translations.delete_all
+          create_translations(audience_ontology, mapping, mappings)
+        end
+      end
 
-    def create_or_fetch_translations(audience_ontology, link, mappings, overwrite: false)
-      translations = TranslatedSentence.for(audience_ontology, sentences_from: self)
-      if translations.any? && !overwrite
-        translations
-      else
-        translations.delete_all
-        create_translations(audience_ontology, link, mappings)
+      def create_translations(_audience_ontology, _mapping, _mappings)
+        query, args = mappings_by_kind_query(self, 'import')
+        imported_ontology_ids = pluck_select([query, *args], :id).reverse
+        imported_ontology_ids.each do |o_id|
+          Ontology.find(o_id).create_translated_sentences
+        end
       end
     end
 
-    def create_translations(audience_ontology, link, mappings)
-      query, args = links_by_kind_query(self, 'import')
-      imported_ontology_ids = pluck_select([query, *args], :id).reverse
-      imported_ontology_ids.each { |o_id| Ontology.find(o_id).create_translated_sentences }
-    end
-  end
+    module Methods
 
-  module Methods
+      def update_or_create_from_hash(hash, timestamp = Time.now)
+        sentence = find_or_initialize_by_name(hash['name'])
 
-    def update_or_create_from_hash(hash, timestamp = Time.now)
-      sentence = find_or_initialize_by_name(hash['name'])
+        sentence.imported   = hash['imported'] || false
+        sentence.text       = hash['text'].to_s
+        sentence.range      = hash['range']
+        sentence.updated_at = timestamp
 
-      sentence.imported   = hash['imported'] || false
-      sentence.text       = hash['text'].to_s
-      sentence.range      = hash['range']
-      sentence.updated_at = timestamp
+        sentence.save!
 
-      sentence.save!
+        execute_sql
+          "DELETE FROM sentences_symbols WHERE sentence_id=#{sentence.id}"
```

Comment:
Lint/Void: Literal `"DELETE FROM sentences_symbols WHERE sentence_id=#{sentence.id}"` used in void context.<br>Style/IndentationConsistency: Inconsistent indentation detected.